src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -12559,10 +12554,9 @@ public static void Main()
     }
 }
 """;
-            CompileAndVerify(source, expectedOutput: "file::member:0");
+            CompileAndVerify(source, expectedOutput: "::Main:9");
```

Comment:
>CompileAndVerify(source, expectedOutput: "::Main:9"); [](http://example.com/codeflow?start=12&length=53)

Are we testing that attributes are actually present/missing in metadata (on the lambda method and the delegate invoke)?